torch/jit/__init__.py

```diff
@@ -1049,7 +1057,77 @@ def _compile_and_register_class(obj, rcb, qualified_name):
     _add_script_class(obj, qualified_name)
 
 
-def script(obj, optimize=None, _frames_up=0, _rcb=None):
+def script(obj, optimize=True, _frames_up=0, _rcb=None):
+    r"""
+    Scripting a function or ``nn.Module`` will inspect the source code, compile
+    it as TorchScript code using the TorchScript compiler, and return a ``ScriptModule``.
+
+    **Scripting a function**
+        The ``@torch.jit.script`` decorator will construct a ``ScriptModule`` with a single ``forward``
```

Comment:
This isn't correct anymore, we don't return a module in this case, we return a function.